src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -220,8 +227,15 @@ class EmojiPicker extends React.Component<IProps, IState> {
         return (
             <div className="mx_EmojiPicker">
                 <Header categories={this.categories} onAnchorClick={this.scrollToCategory} />
-                <Search query={this.state.filter} onChange={this.onChangeFilter} />
-                <AutoHideScrollbar className="mx_EmojiPicker_body" wrappedRef={this.bodyRef} onScroll={this.onScroll}>
+                <Search query={this.state.filter} onChange={this.onChangeFilter} onEnter={this.onEnterFilter} />
+                <AutoHideScrollbar
+                    className="mx_EmojiPicker_body"
+                    wrappedRef={ref => {
+                        // @ts-ignore - AutoHideScrollbar should accept a RefObject or fall back to its own instead
```

Comment:
hmm - I got the linter yelling at me when I used this in js-sdk. Do we allow it in one project but not the other?

Reply:
Err, it should be allowed given we have a heckin' lot of 'em

![image](https://user-images.githubusercontent.com/2403652/94970507-b6cba080-04fc-11eb-971d-1021458a3a04.png)

Does seem to be forbidden in element-web for some reason, there it is often paired with `// eslint-disable-next-line @typescript-eslint/ban-ts-comment`